src/display/editor/editor.js

```diff
@@ -218,11 +218,27 @@ class AnnotationEditor {
     const [tx, ty] = this.getInitialTranslation();
     this.translate(tx, ty);
 
-    bindEvents(this, this.div, ["dragstart", "focusin", "focusout"]);
+    bindEvents(this, this.div, [
+      "dragstart",
+      "focusin",
+      "focusout",
+      "mousedown",
+    ]);
 
     return this.div;
   }
 
+  /**
+   * Onmousedown callback.
+   * @param {MouseEvent} event
+   */
+  mousedown(event) {
+    if (event.button !== 0) {
+      // Avoid to focus this editor because of a right click.
```

Comment:
Maybe tweak the comment slightly, now that the condition was updated above?
```suggestion
      // Avoid to focus this editor because of a non-left click.
```